test/functional/sandbox/html/bogusCommand.prod.html

```diff
@@ -0,0 +1,22 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <title>Alloy Test Page</title>
+        <script>
+            !function(n,o){o.forEach(function(o){n[o]||((n.__alloyNS=n.__alloyNS||
+            []).push(o),n[o]=function(){var u=arguments;return new Promise(
+            function(i,l){n[o].q.push([i,l,u])})},n[o].q=[])})}
+            (window,["alloy"]);
+        </script>
+        <script src="/dist/standalone/alloy.min.js" async></script>
+
+        <script>
+            alloy("configure", {
+                errorsEnabled: false,
+                logEnabled: true
+            });
+            alloy("boguscommand");
+        </script>
+    </head>
+</html>
```

Comment:
From what I can tell just by looking here, it looks like bogusCommand.dev.html is different from bogusCommand.prod.html only in that it loads `alloy.js` vs `alloy.min.js`. Are you planning on reproducing this 
pattern for every test? If so, can we use a querystring parameter and templating engine to produce the script tag that loads the respective alloy file? Or...if we've decided we're going to test the minified build, then I don't think there's any need to test the unminified build. The chance that the minified build would behave correctly while the unminified build would not behave correctly is incredibly low. I like the idea of having the unminified build available for easier debugging, but if that would just require us to come into the html file and manually remove `.min` while we're debugging, that might be sufficient.